configure.ac

```diff
@@ -988,7 +988,23 @@ include <windows.h>
 #include <sys/param.h>
 #endif
   ]])
+  AC_CHECK_MEMBER([struct arpreq.arp_pa],[],[
+  AS_IF([test "x$squid_host_os" != "xwindows"],[
```

Comment:
FWIW, this is the first occurrence of squid_host_os comparison with `windows`. This comparison contradicts our own comment quoted below. I do not know whether this comparison and/or that comment is a problem (or neither). I am just raising a red flag.

```
configure.ac:# on windows squid_host_os is either mingw or cygwin
```

Reply:
fixed. Thanks